### ISSUE TITLE
Stopped working with Minimal Mistakes Theme two days ago

### ISSUE THREAD

**Issue by lsolesen** [eduardoboucas/staticman#183]
Two days ago Staticman v2 suddenly stopped working with my Minimal Mistakes Theme. I opened an issue at the theme https://github.com/mmistakes/minimal-mistakes/issues/1590.

However, it might be connected to this commit eduardoboucas/staticman@28caae6, where you two days ago introduced 'undefined'?

**Comment by mmistakes** [comment]
I'm noticing the same on my personal site which doesn't use this theme, but shares some similar logic. 

Console is printing out:

```
Failed to load resource: the server responded with a status of 500 ()

responseText: "{"success":false,"data":["hidden"],"rawError":{"_smErrorCode":"INVALID_FIELDS","data":["hidden"]},"errorCode":"INVALID_FIELDS"}"
```

**Comment by mmistakes** [comment]
Could it have anything to do with hidden fields and  the introduction of `undefined`? On both the MM and my personal site I use a hidden input for the "honeypot" method of catching spam.

```
<input type="text" id="comment-form-location" name="fields[hidden]" autocomplete="off"/>
```

I tried the [Popcorn demo](http://popcorn.staticman.net/) and it is working fine. Apart from reCaptcha the only other thing I noticed the Popcorn demo not using was `fields[hidden]`.
